### PR TITLE
[AIRFLOW-3005] Replace 'Airbnb Airflow' with 'Apache Airflow'

### PR DESCRIPTION
**Pull request by kaxil**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-3005
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.

### Code Quality

- [x] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

File: tests/sensors/test_http_sensor.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ class FakeSession(object):
     def __init__(self):
         self.response = requests.Response()
         self.response.status_code = 200
-        self.response._content = 'airbnb/airflow'.encode('ascii', 'ignore')
+        self.response._content = 'apache/airflow'.encode('ascii', 'ignore')
```
Comment (feng-tao) [inline tests/sensors/test_http_sensor.py]: should it be apache/incubator-airflow ?

File: tests/sensors/test_http_sensor.py [hunk 2]
```diff
@@ -178,7 +178,7 @@ def test_get_response_check(self):
             method='GET',
             endpoint='/search',
             data={"client": "ubuntu", "q": "airflow"},
-            response_check=lambda response: ("airbnb/airflow" in response.text),
+            response_check=lambda response: ("apache/airflow" in response.text),
```
Comment (feng-tao) [inline tests/sensors/test_http_sensor.py]: same

**Review by feng-tao — COMMENTED** [review]
and have you done a git grep and see if there any other places refer to airbnb/airflow?

**Comment by kaxil** [comment]
@feng-tao Made the necessary changes :)

**Comment by feng-tao** [comment]
@kaxil  lgtm, but you may want to check why CI fails for this change which shouldn't...

**Comment by kaxil** [comment]
Fixed the CI

**Pull request merged**